conf/eslint.json

```diff
@@ -56,6 +56,7 @@
         "no-multiple-empty-lines": [0, {"max": 2}],
         "no-native-reassign": 0,
         "no-negated-in-lhs": 2,
+        "no-negated-instanceof-lhs": 2,
```

Comment:
Can do. What is the process for ensuring that we bump it to 2 on major version release? Do I need to create a new issue to that effect?